client/src/modules/dashboard/Chapters/pages/ChaptersPage.tsx

```diff
@@ -50,28 +60,54 @@ export const ChaptersPage: NextPageWithLayout = () => {
   const isLoading = loading || loadingUser || !data;
   if (isLoading || error) return <DashboardLoading error={error} />;
 
+  const filteredChapters = data.dashboardChapters.filter(({ name }) =>
+    name.toLowerCase().includes(filterChapter.toLowerCase().trim()),
+  );
+
   return (
     <VStack>
-      <Flex w="full" justify="space-between">
+      <Grid w="full" gap="1em" gridTemplateColumns="1fr 2fr 8em">
         <Heading data-cy="chapter-dash-heading" id="page-heading">
           Chapters
         </Heading>
+        <Text srOnly>
+          Type the name of chapter that you are looking for in the filter
+          chapter input to filter out other chapters
+        </Text>
+        {data.dashboardChapters.length > 1 && (
+          <Input
+            width="full"
+            type="text"
+            backgroundColor="gray.50"
+            placeholder="Filter Chapter"
+            value={filterChapter}
+            gridRowStart="2"
+            gridColumnStart="1"
+            gridColumnEnd="-1"
+            onChange={(e) => {
+              return setFilterChapter(e.target.value);
+            }}
```

Comment:
```suggestion
            onChange={(e) => setFilterChapter(e.target.value)}
```
Nit: `onChange` doesn't use the return value, so there's no need to provide it.